src/components/AppHeader.jsx

```diff
@@ -12,24 +19,40 @@ import { useSession } from '@inrupt/solid-ui-react';
 const AppHeader = () => {
   const { session } = useSession();
 
-  const headerStyle = {
-    width: '100%',
-    gap: '20px',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContents: 'center'
-  };
-
   return (
-    <header style={headerStyle}>
-      <h2>Getting Started with PASS</h2>
-      {session.info.isLoggedIn ? (
-        <>
-          <Link to="/PASS/home/">Home</Link>
-          <Link to="/PASS/forms/">Forms</Link>
-        </>
-      ) : null}
-    </header>
+    <>
+      <CssBaseline />
```

Comment:
We already have CssBaseline moved to index.jsx. I don't think we need another one here in Line 24. We can remove this without affecting the styling.

Reply:
Done